config/env_sample.json

```diff
@@ -176,13 +176,45 @@
     "/*cron queries to pull the data from various caliper data sources replace if not relevant with your institution": "*/",
     "RESOURCE_ACCESS_CONFIG": {
         "canvas": {
-            "query": "SELECT 'canvas' AS resource_type, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.object.id'), 35) AS STRING) AS resource_id, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.membership.member.id'), 29) AS INT64) AS user_id, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.group.id'),31) AS INT64) AS course_id, COALESCE(JSON_EXTRACT_SCALAR(event, '$.object.extensions[\\'com.instructure.canvas\\'].asset_name'), JSON_EXTRACT_SCALAR(event, '$.object.extensions[\\'com.instructure.canvas\\'].filename'), JSON_EXTRACT_SCALAR(event, '$.object.name'), 'attachment') as name, datetime(EVENT_TIME) as access_time FROM event_store.events where JSON_EXTRACT_SCALAR(event, '$.edApp.id') IN UNNEST(['http://m.canvas.umich.edu/' , 'http://umich.instructure.com/' ]) and type = 'NavigationEvent' and SUBSTR(JSON_EXTRACT_SCALAR(event, '$.object.id'),24,10) = 'attachment' and JSON_EXTRACT_SCALAR(event, '$.action') = 'NavigatedTo' and JSON_EXTRACT_SCALAR(event, '$.membership.member.id') is not null and SUBSTR(JSON_EXTRACT_SCALAR(event, '$.group.id'),31) IN UNNEST(@course_ids)",
+            "query": [
+                "SELECT 'canvas' AS resource_type, ",
+                "CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.object.id'), 35) AS STRING) AS resource_id, ",
+                "CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.membership.member.id'), 29) AS INT64) AS user_id, ",
+                "CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.group.id'),31) AS INT64) AS course_id, ",
+                "COALESCE(",
+                "JSON_EXTRACT_SCALAR(event, '$.object.extensions[\\'com.instructure.canvas\\'].asset_name'), ",
+                "JSON_EXTRACT_SCALAR(event, '$.object.extensions[\\'com.instructure.canvas\\'].filename'), ",
+                "JSON_EXTRACT_SCALAR(event, '$.object.name'), ",
+                "'attachment'",
+                ") as name, ",
+                "datetime(EVENT_TIME) as access_time ",
+                "FROM event_store.events ",
+                "where JSON_EXTRACT_SCALAR(event, '$.edApp.id') IN ",
+                "UNNEST(['http://m.canvas.umich.edu/' , 'http://umich.instructure.com/' ]) ",
+                "and type = 'NavigationEvent' ",
+                "and SUBSTR(JSON_EXTRACT_SCALAR(event, '$.object.id'),24,10) = 'attachment' ",
+                "and JSON_EXTRACT_SCALAR(event, '$.action') = 'NavigatedTo' ",
+                "and JSON_EXTRACT_SCALAR(event, '$.membership.member.id') is not null ",
+                "and SUBSTR(JSON_EXTRACT_SCALAR(event, '$.group.id'),31) IN UNNEST(@course_ids) "
+            ],
```

Comment:
I think I like this. Ultimately it might make sense to put these queries in separate files, but this makes it a lot more readable! Glad you added the comment to `cron.py` about the multi-line presentation, as it could be confusing initially why the join is necessary.